src/generator.py

```diff
@@ -185,6 +185,8 @@ def create_actors(self, test_case: TestCase):
                 mode = ActorMode.HOST
             elif desc['mode'] == "guest":
                 mode = ActorMode.GUEST
+            elif desc['mode'] == "user":
```

Comment:
As discussed, we should keep some orthogonality between privilege (user vs supervisor) and world (host vs guest)

Reply:
Agree. Should we update the keyword to "host-user" and "host-kernel" then?

Reply:
Having 2 distinct keys in the dict would feel more intuitive to me but as you prefer :+1:

Reply:
done